test/date-time/date-time-comparison.test.ts

```diff
@@ -0,0 +1,1109 @@
+import * as Assert from "assert";
+import { DateTime } from "../../src/date-time";
+import { Duration } from "../../src";
+import { given } from "@nivinjoseph/n-defensive";
+import { IANAZone } from "luxon";
+
+
+suite("DateTime Comparison", () =>
```

Comment:
This suite should be structured by method names

and then in there have these has cases. 
same date 
positive case 
negative case
edge cases